tests/config/pubsub_components.yaml

```diff
@@ -0,0 +1,90 @@
+#
```

Comment:
Can you rename this file to `pubsub_perf_components.yaml` or something like that? As long as it includes the word "perf"